fuzz/Cargo.lock

```diff
@@ -603,6 +603,15 @@ dependencies = [
  "syn",
 ]
 
+[[package]]
+name = "timerfd"
+version = "1.2.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "0bb53e6628675d73224925201a9a41f01c8d31108fdccb983975a1c1449dfc91"
+dependencies = [
+ "libc",
+]
+
```

Comment:
You can get rid of this since you don't use `timerfd` crate anymore.

Reply:
Good catch. Fixed.